assets/javascripts/discourse/controllers/preferences-saved-searches.js

```diff
@@ -0,0 +1,50 @@
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+import discourseComputed from "discourse-common/utils/decorators";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Controller.extend({
+  saving: false,
+
+  @discourseComputed("savedSearches.length", "siteSettings.max_saved_searches")
```

Comment:
we have propertyGreaterThan/propertyLessThan macros available for this kind of cases

Reply:
I did not know about these.